chapters/fa/chapter2/2.mdx

```diff
@@ -428,12 +428,13 @@ model.config.id2label
 - جمله دوم: NEGATIVE: 0.9995, POSITIVE: 0.0005
 
 
-ما با موفقیت سه مرحله خط‌تولید را در اینجا نشان دادیم: پیش‌پردازش توسط توکِنایزرها، گذر ورودی‌ها از مدل و پس‌پردازش! حالا کمی زمان می‌گذاریم تا به صورت عمیق‌تر وارد هر‌کدام از این مراحل شویم.
+ما با موفقیت سه مرحله خط تولید را در اینجا نشان دادیم: پیش‌پردازش توسط توکِنایزرها، گذر ورودی‌ها از مدل و پس‌پردازش! حالا کمی زمان می‌گذاریم تا به صورت عمیق‌تر وارد هر‌کدام از این مراحل شویم.
 
 <Tip>
 
-✏️ **خودتان امتحان کنید!** دو(یا بیشتر) نوشته از خودتان را از خط‌تولید `sentiment-analysis` بگذرانید. سپس مراحلی که در اینجا دیدیم را تکرار کنید و مطمئن شوید همان نتایج را دریافت می کنید!
+✏️ **خودتان امتحان کنید!** دو(یا بیشتر) نوشته از خودتان را از خط تولید `sentiment-analysis` بگذرانید. سپس مراحلی که در اینجا دیدیم را تکرار کنید و مطمئن شوید همان نتایج را دریافت می کنید!
```

Comment:
Fixes:

- there must be a space before parentheses openings.
- "می کنید" -> "می‌کنید"

Suggestion:
"دو نوشته از خودتان (یا حتی بیشتر) را از خط تولید... ... سپس مراحلی را که در اینجا دیدیم تکرار نموده و بررسی کنید که آیا همان نتایج را دریافت می‌کنید!"
instead of:
"دو(یا بیشتر) نوشته از خودتان را از خط تولید... ... سپس مراحلی که در اینجا دیدیم را تکرار کنید و مطمئن شوید همان نتایج را دریافت می کنید!"

Reply:
Applied the fixes.

Went with:
دو نوشته از خودتان (یا حتی بیشتر) را از خط تولید `sentiment-analysis` بگذرانید. سپس مراحلی که در اینجا دیدیم را تکرار کنید و بررسی کنید که نتایج همان هستند!